_inc/client/state/action-types.js

```diff
@@ -119,6 +119,10 @@ export const JETPACK_SEARCH_TERM = 'JETPACK_SEARCH_TERM';
 export const JETPACK_SEARCH_FOCUS = 'JETPACK_SEARCH_FOCUS';
 export const JETPACK_SEARCH_BLUR = 'JETPACK_SEARCH_BLUR';
 
+export const JETPACK_MARKETING_SUBMIT_SURVEY = 'JETPACK_MARKETING_SUBMIT_SURVEY';
```

Comment:
What do you think about calling these `JETPACK_MARKETING_SUBMIT_DISCONNECT_SURVEY`, `JETPACK_MARKETING_SUBMIT_DISCONNECT_SURVEY_SUCCESS`, and `JETPACK_MARKETING_SUBMIT_DISCONNECT_SURVEY_FAIL`? Just to differentiate from any future surveys we do in the plugin.